site/en/docs/extensions/mv3/settings_override/index.md

```diff
@@ -70,22 +73,20 @@ Values in the manifest can be customized in the following ways:
   }
   ```
 
-  All occurrences of the substring "\_\_PARAM\_\_" in the manifest URLs will be substituted with the
-  "install_parameter" value. If "install_parameter" is absent, occurrences of "\_\_PARAM\_\_" are
-  removed. Note that "\_\_PARAM\_\_" cannot be part of the hostname. It needs to occur after the
+  All occurrences of the substring `"\_\_PARAM\_\_"` in the manifest URLs will be substituted with the
+  `"install_parameter"` value. If `"install_parameter"` is absent, occurrences of `"\_\_PARAM\_\_"` are
+  removed. Note that `"\_\_PARAM\_\_"` cannot be part of the hostname. It needs to occur after the
   first '/' in the URL.
 
 ## Reference {: #reference }
 
 An extension can override one or more of the following properties in the manifest:
 
-- **`homepage` (string)** - optional
-
-  New value for the homepage.
+**`homepage` (string)** - optional
```

Comment:
```suggestion
`homepage` (string) - optional
```

Using `**bold**` here may not be necessary here due to the way our styling rules and how Markdown definition list processing works.

The paragraph before a line with a `: ` prefix is wrapped in a `<dt>` tag. These tags are styled with given a higher-than-normal `font-weight` of 600. `<strong>` tags have a font-weight of bold (700). While there _is_ a visible difference in the final rendering, it's minor.

**Rendering in Chrome with `<strong>` inside the `<dt>`**
![screenshot-1](https://user-images.githubusercontent.com/1047763/204959275-910ca3b8-ef12-4dc5-92b6-a6396f672f52.png)

**Rendering  in Chrome with only a `<dt>`** 
![screenshot-2](https://user-images.githubusercontent.com/1047763/204959284-0c38fb3b-ef17-4a57-ac81-64302fbcf69b.png)